### DISCUSSION TITLE
Using reth with polygon

### DISCUSSION THREAD

**Discussion by aathan** [paradigmxyz/reth#3994] [Help] [answered]
I did a `find . -type f | xargs grep -i polygon` and found one single reference in README.md.  I'll go keep looking for docs, but meanwhile, I'd like to ask here about any comments/tips/hints/tricks/cautions regarding using reth with the polygon network. I'm just starting to learn how that ecosystem is put together. Ultimately what I want to do is build a listener that can filter every new block for certain events etc.

**Comment by jmcph4** [comment, marked as answer]
The situation has [greatly improved](https://github.com/paradigmxyz/reth/blob/main/examples/polygon-p2p/src/main.rs).
